config/libc/glibc.in

```diff
@@ -172,6 +172,13 @@ config GLIBC_CONFIGPARMS
       Note: If you need to pass more than one value, separate them with
             '\n'. Eg.:  var1=val1\nvar2=val2
 
+config GLIBC_ENABLE_DEBUG
+    bool
+    default n
```

Comment:
Since technically this would be a change in behavior otherwise, this should be `default y`